ui/app/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -88,23 +86,21 @@ export default class ConfirmPageContainerContent extends Component {
     return (
       <div className="confirm-page-container-content">
         {warning && <ConfirmPageContainerWarning warning={warning} />}
-        {summaryComponent || (
-          <ConfirmPageContainerSummary
-            className={classnames({
-              'confirm-page-container-summary--border':
-                !detailsComponent || !dataComponent,
-            })}
-            action={action}
-            title={title}
-            titleComponent={titleComponent}
-            subtitle={subtitle}
-            subtitleComponent={subtitleComponent}
-            hideSubtitle={hideSubtitle}
-            identiconAddress={identiconAddress}
-            nonce={nonce}
-            assetImage={assetImage}
-          />
-        )}
+        <ConfirmPageContainerSummary
+          className={classnames({
+            'confirm-page-container-summary--border':
+              !detailsComponent || !dataComponent,
+          })}
+          action={action}
+          title={title}
+          titleComponent={titleComponent}
+          subtitle={subtitle}
+          subtitleComponent={subtitleComponent}
+          hideSubtitle={hideSubtitle}
+          identiconAddress={identiconAddress}
+          nonce={nonce}
+          assetImage={assetImage}
+        />
         {this.renderContent()}
```

Comment:
This file looks good. `ConfirmPageContainerContent` is only used in `ConfirmPageContainer`, which is only used in `ConfirmTransactionBase`, the instances of which are never passed a summary component.